### PR TITLE
Room list reflects encryption state

### PR DESCRIPTION
**Pull request by foldleft**
Fixes https://github.com/vector-im/riot-web/issues/11935

Also changes the header icon's lil' shield to match

Screenshot 2020-01-23 at 10 56 06

### CODE REVIEW

**Comment by foldleft** [comment]
note that it's a little more complicated than it perhaps needs to be: I only start watching for state changes such as joins/leaves when I know a room is encrypted.  honestly, probably even on the scale of #matrix:matrix.org that's not likely to be a problem. thoughts?

**Comment by jryans** [comment]
Looking at the design, I think the layered shields on the avatars are the correct size, but the position is a little too far inside the circle. Maybe +1 px down and +1px right?

**Comment by foldleft** [comment]
I think I got it pixel-perfect, it's just that the outline of the shields makes it look like it's up/right by a pixel.  I can move it.

File: src/components/structures/RoomView.js [hunk 1]
```diff
@@ -796,6 +796,7 @@ export default createReactClass({
             return;
         }
 
+        // Duplication betwen here and _updateE2eStatus in RoomTile
```
[the remark this answers is not in the document]
Comment (foldleft) [reply src/components/structures/RoomView.js]: mortifying

**Pull request merged**